pandas/core/panel.py

```diff
@@ -911,7 +911,7 @@ def groupby(self, function, axis='major'):
         -------
         grouped : PanelGroupBy
         """
-        from pandas.core.groupby.groupby import PanelGroupBy
+        from pandas.core.groupby.panel import PanelGroupBy
```

Comment:
I would make this importing just from `pandas.core.groupby`

Reply:
done